database/sinkdb/op.go

```diff
@@ -69,3 +69,11 @@ func AddAllowedMember(addr string) Op {
 		}},
 	}
 }
+
+// Error returns an Op representing an error condition.
+// Exec will return err, and have no effect,
+// when the returned Op is executed.
+// If err is nil, Error returns the zero Op.
```

Comment:
I am a leeeettle bit worried that this is non-obvious but maybe it's part of a pattern I'm not familiar with.